### DISCUSSION TITLE
KDB.AI

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#103] [VDB Comparison]
Hey everyone, this is the start of discussions related to [KDB.AI](http://kdb.ai/) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by sancoap** [comment]
Hi Arunesh,
We recently released KDB.AI 1.1 and I would like to update the KDB.AI row with the latest capabilities. What's the process for this?
Thanks,
Sandeep

**Reply by AruneshSingh** [reply]
Hey @sancoap, you can read about the file structure [here](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) and then modify the relevant attributes for [KDB.ai](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/data/kdbai.json) and raise a PR. Our table maintainers will help you with any clarifications/modifications on the PR then. Once approved, the data will be merged and automatically be updated on the website.
